v2/our-platform/api-reference/project-management.rst

```diff
@@ -113,9 +113,9 @@ Returns a Project Response object, containing the newly created Project object.
     {
       "projects": [{
         "projectId": 987654321,
-        "name": , "My First Project"
-        "created": "09/30/2020",
-        "modified": "10/01/2020",
+        "name": , "My-First-Project"
```

Comment:
Revert the dashes, please. I removed them on purpose

Reply:
This is also still a json bug; `"name": , "My-First-Project"` comma goes on the end